examples/ctrldeploy-ctrlreg.yaml

```diff
@@ -5,22 +5,52 @@ metadata:
wt2vpHv5ify3f3l+QhiICsiwL5FA/oFJD8WoVUduGLvHwIgbhmTEHG/kOTCvcBzTMSNjPmUihOmQiInAk6hMCey5gMX3Ak9pG5EPJ2WVSjTyEvXbZForhTRgNllEl0XIKblMwvNI6a5d8pb5AXEw9hiRgouCCSYjeSwcJisBDYE/EL0kGR7s/SYJaCQS/H/MUaWSmkdAN1EakihNU66qVg3UDE0Zn+WaVWtb1XLpRzvmgWB9/FcMbIYqcIun0Nh9/e80G9Vi/T8EbKv/SHgc4trc8amUO2YEG9f/Vluv/61qp9qu1Y6q9Ua93irW/0NAsf4X6/8h1n8JgnZjHxBZd/+UlsdPISO480LXJtdJaDnD0FIKmKIuVdQuEWIyhrUhqjSlfgydatlSUhr7fEj9czaisa9sMqK+ZPczjHJKkuhwlk0/yx0ySvl8xforJgXbxn90e1AsdRwehzuWhJvrv/oi/lfbGP8bUBIW8f8AUMT/Iv4fqP4z0f7GBJKuCSQ7h3vTS4LioOvXr8QaMJ9RIHeZPiZ//AG9fDpkvkSsBPVl3cVDmAeDUIZrzzaUfrRnHga2jf+fJ8wHC5CWinbeC9wQ/6uNts7/q7CGN6odjP9YABbx/xDw9WuFEJeNvBBCsxeAy5dJRXsQtngjMqHyWkD7F1KWE1pvte0ysT5g0gWehP0tRcckGwF1YqhGpPyL/PcvcrmnYBGXnuJivg4F+C1bhdDeG2Ho4k3u8kdL/eeBbf1fDKmz74uADf7fbDdai/yv1UH/rzeL/O8gUOR/Rf73vfK/SqVyLwfUMYTGagK19+/Gtpb3As588EcmBtxnK1LDLEZJK41bluPz2LU3JY1PTwdXj9TvJsJVuSguMyIG27FLFRjovRE8jjQHeBtJuMheZpinLot8Pg9YqPKNOq81HZB0BVBpF0SO4O8waRszpX99sCN9MaPKmeirSF/txcRmAg/QPqqjh7Qco+w9CVE3MUfBxug8eYN6SCuI0VLD8YwNJ5zfOTwceePYDNqOAbyKIzBIZpgXLL18RMA7SMINpWAOF+6essjQbaRkXpu52dP79Mqvyg+Rl8sP0WRF2Pe3kkXXpdtTcD0Vb6+8aFmNW83UWEpAI0NeMlC8SliZPnSTlRJ0OKjWC9dbqA4dS9gSI3taIH0ND8Dy/wrxFGYzYCMkm0p4jTCg18NF5alTl/EQ3yrrqL5yQ+M5tzEOkP9tm/9nblDRBdYUZMBFBZMDTHfY2tJgU/3fqifvf2tQ/7ew/ocqoMj/DwJJlX+vev6glXuV6hYtceVW3pm2iQsa7bqLt9qGDri3l/Kq+ZCf75uxTb6VkPo6kXyDRB/wK9L8k+8sbOv/i7xw912ADf4PZX/D+H+n2mzWG+D/9WanWfj/IaCo/4v6/xDvf7DaXGSn51k0+YlfAKEwbJ2nTz1sewsq4mL+Dmdtk6pu0YYgDSPJepE8PMNs0LAigUNcPgwzAZYh73Lc7Z9Bp+E5QZyTpMYaglcZ9aSPsl39Tet92h3cw7mTcXC6x3qupeKFUOW7jLzQ+/jkH9ZtwrP1GpR2TcGfy1sll+WXxIJl17yDAI7yXObWX/PAvyfep5QoqREgRPmDKpdbIkD15wuE7ceBVDCigQelDFS29BADWpb3lJC+4LHywtPPrmPfv+Zgt/N7tmxepkRZY34cC6d7m1U6kf5F903vQ+/s9mrw+epDb/Bx0L/tZb2gFDaHe071WeBcnna6MnFbtoclC8nxZwxOrmIJ4umv/Tc369lZWH6CaQ1pWL4CiJd2jtJ6b8o6TrkfB+wCreZxD14zlWXeIEQgLnS6XSexp34fy/NX8rOs41w/wah7FfpgnErE7HG/NxLLhLVJVI8JykkLGzvHwzZ1zX1ED9jbSYrbyXBvVjepxjXn5y64C6ia9erDOf3o1LWAZ4Dd938SG9+hDNy0/2PO/2H9V4PrFp7/a3c6Rf13CHg8RD7zls8iNB56kwenuDy/b1CrfQIDhosw2cmp/8l3cvaD9f4/NVJ74gegm85/tNrG/xu1Vr3abuH3P+1a4f8HgWL/p9j/+V77P/mdEJvUSiVTkZqNk+Tcnk187lB/wqWyW9VqbWNppOgYxmBOgu/aolzd2vVndC5LpePjY3J7dX5lEzLoXUAdR4YMJshIwMQYZwJqCii+EdRd/wMRAOXNvlAQEEPLmfCZSbtxomQ5Tccs3YEBQ4bfmE49F1SGAky+U9UCMYM0/uRDFFM8alXr05auNm1d81hjTsAS75AgmBcLhgxf2CatSa1rKlw9NxAWM+ikri49pYzxoVjBR8Bds4yelHnIsGoCx0FJI3PllZjJtV6HicuNN4aIGhdGpelVCFgyAZMCCUElZkwGlACMnrpQSPg80r2jGCIHOBKa2aoCB9+uoCMa2ra+3mpfo5KXb+LLeWM4TirF/SzrIfbtmIJJhOr5uTFoN5ty3pYxj5tBjKSh/mza+PI8LeRSg+E6LKgJVWQYez7oD2ODy2ehzymasjRuMBI8AP61TZSQC/K6R97f9M7x8mzQ6972kr2Si+61tobFZ9hULlEFBNjDZF/I3JD9jvHexa4UWJ7jQzRaAe6U9sMh+OG3nMDaiziS7M0YzYjzWmpMxLC74MB8PJ5LX0jazxPg7j4ed06eSAZoXf0oWyMkDzAYQjKaPAJi9d2JQTgnM5S0Pkc9BbWBSDxDyRTq4GjJMZqMTmNBJ/FAm5Q3USxnI1BCdnKXZuZlvazCP4hXEMbLWTOQs8mQigX15oK6iTVmIO6GDfrnvUyvkdkP1Hw94kWGy3wOmSA2YxdMdh3lTYFNxaGFj+cVmqyaBtKcY7A4L5M2EXJ2/d4mnVY1SJ7B+DETix7nbBiDV6ZnbdL/NuDEwZX/xHBykrXilqvgKZ9uHEQLTKb0sWF9GNnWq5MRrDz2ixdn+oTOy28vBiwA+8WLOIS/7/URI7i4gfRS6V8evXxpvfqb1BUFFFBAAQUUUMDPC/8HdT8zjgBQAAA=
```

Comment:
Missing copyright notice.

Reply:
This file is generated from script - I'll add license notice to to generation script